examples/barotropicqg_betaforced.jl

```diff
@@ -69,7 +68,7 @@ seed!(1234) # reset of the random number generator for reproducibility
 nothing # hide
 
 # Next we construct function `calcF!` that computes a forcing realization every timestep
-function calcFq!(Fh, sol, t, cl, v, p, g)
+function calcFq!(Fh, sol, t, cl, vs, pr, gr)
```

Comment:
We may want to encourage a more verbose coding style in the future. But this is fine for now of course!